view/keys.go

```diff
@@ -0,0 +1,51 @@
+package view
+
+import "github.com/charmbracelet/bubbles/key"
+
+type keyMap struct {
+	Up     key.Binding
+	Down   key.Binding
+	Delete key.Binding
+	Help   key.Binding
+	Quit   key.Binding
+	Exit   key.Binding
+}
+
+func (k keyMap) ShortHelp() []key.Binding {
+	return []key.Binding{k.Help, k.Delete, k.Quit}
+}
+
+func (k keyMap) FullHelp() [][]key.Binding {
+	return [][]key.Binding{
+		{k.Up, k.Down},
+		{k.Delete, k.Help},
+		{k.Quit, k.Exit},
+	}
+}
+
+var keys = keyMap{
+	Up: key.NewBinding(
+		key.WithKeys("up", "k"),
+		key.WithHelp("↑/k", "move up"),
+	),
+	Down: key.NewBinding(
+		key.WithKeys("down", "j"),
+		key.WithHelp("↓/j", "move down"),
+	),
+	Delete: key.NewBinding(
+		key.WithKeys("d", "delete"),
+		key.WithHelp("d/delete", "delete dose"),
```

Comment:
Hitting `d` is easy to do by accident, and there is no real recourse to restoring a dose. Replacing that with `ctrl+d` should reduce accidental deletions.
```suggestion
		key.WithKeys("ctrl+d", "delete"),
		key.WithHelp("ctrl+d/delete", "delete dose"),
```